lib/iris/fileformats/pp.py

```diff
@@ -1403,9 +1438,11 @@ def save(self, file_handle):
         # the data itself
         if lb[self.HEADER_DICT['lbpack'][0]] == 0:
             data.tofile(pp_file)
+        elif lb[self.HEADER_DICT['lbpack'][0]] == 1:
+            pp_file.write(packed_data)
         else:
             msg = 'Writing packed pp data with lbpack of {} ' \
-                'is not supported.'.format(lb[self.HEADER_DICT['lbpack'][0]])
+                  'is not supported.'.format(lb[self.HEADER_DICT['lbpack'][0]])
             raise NotImplementedError(msg)
```

Comment:
I'm not sure about this exception. I'd look at implementing either a ValueError (generic), or an ImportError (as it means we don't have the necessary package).